src/helpers.jl

```diff
@@ -2,6 +2,48 @@
 # Based on the particle Python package implementations from the SciKit-HEP
 # group: https://github.com/scikit-hep/particle/blob/master/src/particle/pdgid/functions.py
 
+const _common_alias = Dict(
+    "mu" => ["muon", "mu-"],
+    "e" => ["electron", "e-"],
+    "tau" => ["tauon", "tau-"],
+    "gamma" => ["photon", "γ"],
+    "H" => ["higgs"]
+)
+"""
+    Particle(name::String)
+
+Construct particles from PDG name.
+
+Or commonly alias, alias do not cover all particles.
+"""
+function Particle(n::String)
+    # determine if anti particle
+    anti = false
```

Comment:
Should this anti flag also check for +/-, e.g. `"mu+"`, as this naming is also used in `_common_alias`?

Reply:
I'm trying to think is `blah+` only mean anti-particle for leptons? I guess not because there's `pi+` etc.

Maybe I can make `mu~ => ["mu+"]` alias?

Reply:
Yeah that's an argument, to be honest I just thought about `"mu+"` as example, as it was present in `_common_alias`. 
Sounds not to bad to solve it via an alias :+1:

Reply:
done!